documentation-site/pages/components/spinner.mdx

```diff
@@ -111,3 +112,9 @@ If you want to customize the default styling of the spinner you can extend it us
 </Example>
 
 In this example we modify the size of the spinner (via `width`, `height`, and `borderWidth`) as well as the highlight color (via `borderTopColor`). You could also use `withStyle` to add margins to the spinner or align it within another element.
+
+<Example title="$as" path="spinner/next-span.js">
+  <SpinnerNextSpan />
+</Example>
+
+By default the spinner styles a single `div` element. You can change the underlying element type by using the `$as` prop. In this example we change it to a `span` and add `display: block`. This allows us to use the `Spinner` inside of a `p` element. Note, we could also set `display: inline-block` to prevent the spinner from wrapping to a new line.
```

Comment:
While helping someone who brought up the issue mentioned here ^ I also notices that the API for this styled element is different from the current `Spinner` component but we didn't document it anywhere. Could you add it here as well?